android-components/components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -938,6 +938,16 @@ sealed class TranslationsAction : BrowserAction() {
         val translationError: TranslationError,
     ) : TranslationsAction(), ActionWithTab
 
+    /**
+     * Indicates an app level translations error occurred and to set the error on
+     * [BrowserState.translationEngine].
+     *
+     * @property error The error that occurred.
```

Comment:
```suggestion
     * @property error The [TranslationError] that occurred.
```
We could reference the class where possible if the naming and wording just makes sense.

Reply:
Thanks, good idea, updated!